src/Chainweb/Payload/PayloadStore.hs

```diff
@@ -18,170 +18,20 @@
 --
 module Chainweb.Payload.PayloadStore
 (
--- * Transaction Database
-
-  BlockTransactionsStore(..)
-, BlockPayloadStore(..)
-, TransactionDb(..)
-, TransactionDbCas
-, transactionDbBlockTransactions
-, transactionDbBlockPayloads
-
--- * Caches
-
-, OutputTreeStore(..)
-, TransactionTreeStore(..)
-, BlockOutputsStore(..)
-, PayloadCache(..)
-, PayloadCacheCas
-, payloadCacheBlockOutputs
-, payloadCacheOutputTrees
-, payloadCacheTransactionTrees
-
--- * Payload Database
-
-, PayloadDb(..)
-, PayloadCas
-, payloadCache
-, transactionDb
+  module Chainweb.Payload.PayloadStore.Types
```

Comment:
This module has a single function in it now. Why create a types file?

Reply:
The single function needs `Chainweb.BlockHeader.Genesis (genesisBlockPayload)`, and this is the thing that's causing our circular dependency. (`Chainweb.BlockHeader.Genesis` depends on pact but pact now depends on payload store)